comma.py

```diff
@@ -0,0 +1,8 @@
+def comma(s1):
+    for f in range(len(s1)):
+        if s1[f] not in "0123456789,":
+            print("NaN")
+            #return
+        else:
+            print(1) 
+comma("06,,4")
```

Comment:
looks like this part is not complete and this will not run with two commas, can you fix it then I will approve